### ISSUE TITLE
hero_replaceViewController back button

### ISSUE THREAD

**Issue by leobouilloux** [lkzhao/Hero#134]
Hello,

I'm new using Hero on my project.
I have implemented the TableView to CollectionView in my project by taking a look to the sample project.
My application is a file browser.
Let's consider the following situation:
[VC1] -> [VC2] -> [VC3]

If I do VC3.replaceViewController, I lost my back button.
Is there any parameter/workaround known to keep the backbutton?

Thank you very much for your work!

**Comment by lkzhao** [comment]
Might be a bug. Let me check this out when I have time.

**Comment by Manny7311** [comment]
Same here, I tried hero_dismissViewController() but that's not working either

**Comment by LeonardoGtz** [comment]
I'm having the same problem, did you find a solution?

**Comment by leobouilloux** [comment]
Hello,

Forgot to mention that I'm using Material design on my app using [Material framework](https://github.com/CosmicMind/Material).

I have absolutely no idea if this bug is effective using the built in components.

@LeonardoGtz 
The only way I found to make it work is to hide the navigationBar contained in the navigationController and to create a new bar manually. Tricky for sure, but it's the only solution I've found.
